### PR TITLE
sourmash/index stub, nf-test, snap

### PR DESCRIPTION
**Pull request by sateeshperi**
need help with checking the contents of `.zip` file in nf-test/groovy

### CODE REVIEW

File: modules/nf-core/sourmash/index/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                """
+                input[0] = SOURMASH_SKETCH.out.signatures
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.signature_index.collect { file(it[1]).getName() } +
```
Comment (mashehu) [inline modules/nf-core/sourmash/index/tests/main.nf.test:63]: we had a md5sum for `path: output/sourmash/test.sig` in pytest. can you add this also in the nf-test snapshot?

**Review by mashehu — DISMISSED** [review]

**Comment by nvnieuwk** [comment]
Maybe this can help? https://subscription.packtpub.com/book/programming/9781849519366/4/ch04lvl1sec54/reading-data-from-a-zip-file

File: modules/nf-core/sourmash/index/tests/main.nf.test (hunk 1 above)
Comment (SPPearce) [reply modules/nf-core/sourmash/index/tests/main.nf.test]: This is the output from the `SOURMASH_SKETCH` setup process, so not required here.

**Review by nictru — APPROVED** [review]
LGTM

**Pull request merged**